app/models/manageiq/providers/ibm_cloud/inventory/collector/vpc.rb

```diff
@@ -115,7 +115,15 @@ def volume_profiles
   end
 
   def tags_by_crn(crn)
-    vpc.cloudtools.tagging.collection(:list_tags, :attached_to => crn, :providers => ["ghost"]).to_a
+    retried = false
+    begin
+      vpc.cloudtools.tagging.collection(:list_tags, :attached_to => crn, :providers => ["ghost"]).to_a
+    rescue IBMCloudSdkCore::ApiException
+      raise if retried
+      sleep(5)
```

Comment:
Minor but rubocop is right
```suggestion
      raise if retried
      
      sleep(5)
```